.github/workflows/test-erlang-otp-22.3.yaml

```diff
@@ -18,9 +18,9 @@ jobs:
       - name: CONFIGURE OTP & ELIXIR
         uses: actions/setup-elixir@v1
         with:
-          otp-version: 21.3
+          otp-version: 22.3
           # https://github.com/elixir-lang/elixir/releases
-          elixir-version: 1.8.0
+          elixir-version: 1.10.3
```

Comment:
This has changed in multiple place. [This tells me that we are still on 1.8](https://github.com/rabbitmq/rabbitmq-common/blob/d4efb0a4a243a1cad326f7f01e6bb006344916d3/mk/rabbitmq-github-actions.mk#L9). Can you confirm @dumbbell?

Reply:
@gerhard Looks like that was pollution from my local env. I've regenerated the files. Thanks for pointing it out. I noticed it in the diff but didn't realize it was unintended.